usagov/vote-gov-drupal
# VOTE-1116 Create the revision date format and translations

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-1116)

## Description (optional)
Last updated date translated for each language

## Deployment and testing (required, if applicable)

### Post-deploy steps
1. lando retune

### Testing steps
1. Visit a state page
2. View the state page in each language and confirm the 'Last Updated Date' format and translations display correctly.
